libraries/anvill_passes/tests/src/RecoverStackFrameInformation.cpp

```diff
@@ -61,6 +62,11 @@ TEST_SUITE("RecoverStackFrameInformation") {
         CHECK(RunFunctionPass(module.get(),
                               CreateRecoverStackFrameInformation(
                                   *error_manager.get(), lift_options)));
+
+        if (init_strategy ==
```

Comment:
Can you comment on why we're printing the module out only for this strategy?

Reply:
This was temporary code to debug the pass, and I have now removed it